cypress-custom/integration/fee.test.ts

```diff
@@ -45,6 +53,9 @@ describe('Fee endpoint', () => {
 
 describe('Fetch and persist fee', () => {
   beforeEach(() => {
+    // set the Cypress clock to now
+    // only override Date functions
```

Comment:
Is this a reset? Cause i think they already do that (they call clock.restore() in each test)
if im not mistake, u are changing the now value, and setting it to also the current time. So I'm not sure I understand this...

Reply:
this is not how it works, no. They check for all time based functions and wrap them. feel free to run the tests without this reset. it won't work, as we rely on setTimeout and clearInterval in different ways that are too complicated to try and fix here. We are specficying `Date` here as the only function we want wrapped. For all tests.